modules/stanford_news_extras/stanford_news_extras.module

```diff
@@ -30,3 +29,71 @@ function stanford_news_extras_block_view_alter(&$data, $block) {
     $data['title'] = '<none>';
   }
 }
+
+/**
+ * Handle tasks for deprecating the image credits.
+ *
+ * - Don't display the credits field unless there is text in that field.
+ * - Update all the appropriate title and description fields to handle
+ *   deprecation.
+ *
+ * @param array $form
+ *   The form to update.
+ * @param string $form_id
+ *   The form ID. Make sure we're making changes to the right form!
+ */
+function stanford_news_extras_deprecate_image_credits(&$form, $form_id) {
```

Comment:
Since it is in the news_extras module, hopefully it won’t be a problem.

I pushed up the stanford_jse_dept.info  and stanford_soe_helper.info with the dependencies for news_extras.